source/libsodium/crypto_core_salsa208.d

```diff
@@ -10,7 +10,7 @@ module libsodium.crypto_core_salsa208;
 
 @nogc nothrow:
 
-import libsodium.export_;
+import libsodium;
```

Comment:
That doesn't seem right

Reply:
I am not found best way to implement it by script: dstep isn't imports some important `#include`. But `libsodium` module contains all symbols of package as public imported.

In any case, if the names match, a compilation error will occur and we will not miss this.

Reply:
And this creates a circular dependency in the imports, which is the reason why it was done manually in the first place.

Reply:
Yes, but circular dependency has not a problem in D, except if module constructors is used (what never occurs in C bindings)